tests/unit/App.spec.ts

```diff
@@ -27,7 +27,10 @@ vi.mock('@cornerstonejs/core', () => {
           })
         }
       }
+      // Disable eslint rule for this function
+      /* eslint-disable @typescript-eslint/no-empty-function */
       enableElement() {}
```

Comment:
This should make linter happy
```suggestion
      enableElement: vi.fn()
```